ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -439,9 +439,10 @@ std::tuple<ttnn::Shape, ttnn::MemoryConfig, bool, bool> get_conv_padded_input_sh
         }
     }
 
-    bool use_non_tile_height = shard_layout == TensorMemoryLayout::HEIGHT_SHARDED && out_channels <= 256 && conv_config.act_block_h_override == 0 &&
-        conv_config.dtype == DataType::BFLOAT16 && conv_config.output_layout == Layout::ROW_MAJOR;
-    use_non_tile_height = use_non_tile_height && conv_config.input_channels_alignment != 16;  // shalow conv varient
+    // shallow conv variriant not supported
+    // out_channels <= 256 incorrect output from pack_untilize_dst if output > 256 Tracking --> #14236
+    bool use_non_tile_height = (shard_layout == TensorMemoryLayout::HEIGHT_SHARDED) && out_channels <= 256 && conv_config.act_block_h_override == 0 &&
+        conv_config.dtype == DataType::BFLOAT16 && conv_config.output_layout == Layout::ROW_MAJOR && conv_config.input_channels_alignment != 16; //shalow conv varient
```

Comment:
can you put a comment pls for why is only bfloat16 supported. I guess that we don't support bfp formats since they have to be tiled?
And I assume there is not fp32 because it's currently unsupported by the conv ( but should be in feature?)

Reply:
Yes, FP8 needs to be tiled. I'll try adding FP32. I noticed FP32 is used in the activation_type for test_resnet50_conv_wh_fp32.